tests/PHPStan/Analyser/data/bug-8620.php

```diff
@@ -0,0 +1,33 @@
+<?php declare(strict_types = 1);
+
+namespace Bug8620;
+
+use function PHPStan\Testing\assertType;
+
+class HelloWorld
```

Comment:
To have duplicate classes in the same namespace is not acceptable. Use a single file at tests/PHPStan/Rules/DeadCode/data/bug-8620.php and just add assertType calls into it.